model/generators/accounts.py

```diff
@@ -112,36 +114,14 @@ def get(self, account_id) -> Account:
         assert account is not None, f"No account with id: {account_id}"
         return account
 
-    @property
-    def total_supply_celo(self) -> float:
-        """
-        sums up celo balances over all accounts
-        """
-        return self.floating_supply.celo + self.reserve.balance.celo
-
-    @property
-    def tracked_floating_supply_celo(self) -> float:
-        """
-        sums up celo balances over all accounts except reserve
-        """
-        return reduce(lambda s, account: s + account.balance.celo, self.accounts_by_id.values(), 0)
-
-    @property
-    def tracked_floating_supply_cusd(self) -> float:
-        """
-        sums up cusd balances over all accounts except reserve
-        """
-        return reduce(lambda s, account: s + account.balance.cusd, self.accounts_by_id.values(), 0)
-
     @property
     def tracked_floating_supply(self) -> Balance:
         """
         Tracked floating supply which originates from
         """
-        return Balance(
-            celo=self.tracked_floating_supply_celo,
-            cusd=self.tracked_floating_supply_cusd
-        )
+        return reduce(lambda supply, balance: supply + balance,
+                      map(lambda account: account.balance, self.accounts_by_id.values()),
```

Comment:
I think list comprehension is the preferred way but sometimes I forget and start thinking more functional. I'll refactor these for list comprehension tho' because I think PEP8 wise that's preferred. 👍

Reply:
actually I think I can turn that into a `sum`